modules/migration/schemas/issue.json

```diff
@@ -0,0 +1,114 @@
+{
+    "title": "Issue",
+    "description": "Issues associated to a repository within a forge.",
```

Comment:
Off-topic: what is the definition of `forge` in this specific context?

Reply:
Examples were added for clarification.